src/codechicken/nei/recipe/GuiRecipeTab.java

```diff
@@ -177,7 +181,13 @@ public static void loadHandlerInfo() {
         NEIClientConfig.logger.info("Loading handler info from " + (fromJar ? "JAR" : "Config"));
         handlerMap.clear();
         URL handlerUrl = Thread.currentThread().getContextClassLoader().getResource("assets/nei/csv/handlers.csv");
-        
+        handlerMap.putAll(handlerAdderFromIMC);
```

Comment:
Switch these up.  Remove first, and then add.  The expected use case was someone might want to remove a default handler, And someone else might want to add a replacement handler.